corehq/apps/domain/views/accounting.py

```diff
@@ -693,7 +693,7 @@ def dispatch(self, request, *args, **kwargs):
         return super(CreditsWireInvoiceView, self).dispatch(request, *args, **kwargs)
 
     def post(self, request, *args, **kwargs):
-        emails = request.POST.get('emails', []).split()
+        emails = request.POST.get('emails', '').replace(' ', '').split(',')
```

Comment:
why did we choose to comma separate here in the first place? was there an issue with that?

Reply:
It was space separated before. Ops requested it be comma separated.

Reply:
got it. fine with that then!